crates/oracle/src/runner/jrpc_utils.rs

```diff
@@ -65,9 +66,11 @@ where
         let op = move || {
             trace!(?id, ?request, %network, "Sending JRPC call");
             let start = std::time::Instant::now();
-            let result = transport
-                .send(id, request.clone())
-                .map_err(backoff::Error::transient);
+            let network2 = network.clone();
+            let result = transport.send(id, request.clone()).map_err(move |e| {
+                METRICS.track_jrpc_failure(network2.as_str());
+                backoff::Error::transient(e)
+            });
```

Comment:
Had to put an `Arc` over the network type to prevent excessive cloning of `Strings`. 

Could this be achieved without that?